### ISSUE TITLE
Create video(s) for upcoming sale or use web page

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#2241]
Users need updated sale or use information. In #2115 we drafted it. In this issue we will finalize a video that will go with this new page, and circulate both the new page and video(s) to internal offices for review. In #2225 we will post the new page and video(s) once edits received.

**To do**
- [ ] Create script and video for the article (Comment from Dorothy: This will be done by @jjonesfec and other Information Division staff)
- [ ] Circulate article from #2115, the video(s) and the video scripts for review by RAD, OGC, Public Records and content team

**Comment by dorothyyeager** [comment]
My understanding is that script creation and video-making is underway.

**Comment by dorothyyeager** [comment]
The video drafting is in progress but blocked due to vacations until 9/4.

**Comment by dorothyyeager** [comment]
The scripts and article were circulated today. The videos are in production, but it makes sense to have the scripts reviewed first.

**Comment by dorothyyeager** [comment]
We received OGC's comments today and RAD's a week ago, but are still waiting on Public Records. It's likely that this issue will continue into next sprint.

**Comment by dorothyyeager** [comment]
We have received Public Records' comments. Extensive revisions are required. This definitely will continue into the next sprint.

**Comment by dorothyyeager** [comment]
We've circulated the revised article and video scripts to the stakeholders for their review and George spoke with Public Records to address their suggestions. Once the reviewing offices sign off, we'll be able to move on creating the videos.

**Comment by dorothyyeager** [comment]
Heard from RAD but not OGC yet, so will move to 7.3

**Comment by dorothyyeager** [comment]
OGC was sent another reminder yesterday.

**Comment by dorothyyeager** [comment]
We've notified OGC that absent any objection from them, we will go ahead and proceed with the videos and the related Record article (since the Record article is routine website work, we'll update #2225 once it's up; the videos will be embedded in it).

**Comment by dorothyyeager** [comment]
OGC has let us know that they do want to provide some feedback today. We will incorporate their feedback into the videos and article. Future work on this issue will take place in #2225 when we are ready to post the article and videos.
